src/components/Input.tsx

```diff
@@ -6,42 +6,62 @@ import { AriaLiveErrorMessage } from './AriaLiveErrorMessage';
 type InputProps = {
   register: any;
   type?: string;
-  name: 'name' | 'email';
+  name: string;
+  label: string;
   placeholder?: string;
   errors: FieldErrors;
-  watchedValue?: string;
+  watchedValue?: string | number | Date;
   className?: string;
+  isEventInput?: boolean;
 };
 
 export const Input: React.FC<InputProps> = ({
   register,
   type = 'text',
   name,
+  label,
   placeholder,
   errors,
   watchedValue,
   className = '',
+  isEventInput = false,
 }) => (
-  <label className="relative mb-10 w-full">
-    <span className="absolute -translate-y-5 text-xs text-dark-gray">
-      {placeholder}
+  <label
```

Comment:
check https://www.ovl.design/text/inclusive-inputs/, section `Association of labels and inputs`.

> Modern screen readers have robust support for the implicit pattern, but some voice control programs do not support it. Because of that, it is good practice to always be explicit. You can see updated [support data for the computation of accessible names](https://a11ysupport.io/tech/html/label_element#support-table-0) on a11ysupport.

it is quite outdated, so feel free to prove me wrong